aws/resource_aws_db_option_group.go

```diff
@@ -273,12 +273,24 @@ func resourceAwsDbOptionGroupUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 
 		log.Printf("[DEBUG] Modify DB Option Group: %s", modifyOpts)
-		_, err = rdsconn.ModifyOptionGroup(modifyOpts)
+
+		err = resource.Retry(5*time.Minute, func() *resource.RetryError {
+			var err error
+
+			_, err = rdsconn.ModifyOptionGroup(modifyOpts)
+			if err != nil {
+				if isAWSErr(err, "InvalidParameterValue", "SQLSERVER_BACKUP_RESTORE") {
```

Comment:
Two things:
* Can you please add a comment with the full error message?
* Can you adjust the string checking for the relevant IAM part, instead of just the parameter in question? We don't want to retry on non-IAM messages for `SQLSERVER_BACKUP_RESTORE` and we also would love to proactively catch IAM issues for other parameters if we can.

e.g.

```go
// InvalidParameterValue: IAM role ARN value is invalid or does not include the required permissions for: SQLSERVER_BACKUP_RESTORE
if isAWSErr(err, "InvalidParameterValue", "IAM role ARN value is invalid or does not include the required permissions") {
```